server/jobs/reruns.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+import dotenv from 'dotenv';
+import db from '../models';
+
+import { getOrCreateProject, assignProject } from '../modules/freckle/projects';
+import { accessChannel } from '../modules/slack/slackIntegration';
+import {
+  sendDevOnboardingMail, sendSOPOnboardingMail, sendITOffboardingMail, sendSOPOffboardingMail,
+} from '../modules/email/emailModule';
+import { executeEmailAutomation } from './helpers';
+import { findPartnerById } from '../modules/allocations';
+import { retryAutomations } from '../modules/automations';
+
+dotenv.config();
+const { SLACK_AVAILABLE_DEVS_CHANNEL_ID, SLACK_RACK_CITY_CHANNEL_ID } = process.env;
+
+const {
+  SlackAutomation, FreckleAutomation,
+} = db;
+
+const slackAccessChannels = async (slackAutomation, status, existingPlacement, automationId, channelId, context) => {
```

Comment:
Could you rename this function to something like `retryAccessChannel` to avoid it being confused with `accessChannel` function calls

Reply:
resolved